src/sidebar/analytics.js

```diff
@@ -50,8 +50,27 @@ function analytics($analytics, $window, settings) {
      *  in our analytics. Example: 'sidebarOpened'. Use camelCase to track multiple
      *  words.
      */
-    track: function(event){
-      $analytics.eventTrack(event, options);
+    track: function(event, label, metricValue){
+      $analytics.eventTrack(event, Object.assign({}, {
+        label: label || undefined,
+        metricValue: isNaN(metricValue) ? undefined : metricValue,
+      }, options));
+    },
+
+    events: {
+      ANNOTATION_CREATED: 'annotationCreated',
```

Comment:
👍 For having it clear in one place what events we are recording and making it easy to grep for usages.